src/main/java/core/basesyntax/dataprocessor/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.dataprocessor;
+
+import core.basesyntax.colors.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
The idea for improvement:  instead of using a temporary variable - use calculation directly 
```suggestion
        return Color.values()[random.nextInt(Color.values().length];
```